python/example_code/lookoutvision/datasets.py

```diff
@@ -0,0 +1,282 @@
+                    manifest = Datasets.create_json_line(image_path, 1, dttm)
+                    mfile.write(json.dumps(manifest) + "\n")
+                # create json lines for normal images
+                response = s3_client.list_objects_v2(
+                    Bucket=bucket, Prefix=prefix + "normal/", Delimiter="/"
+                )
+                for file in response["Contents"]:
+                    image_path = "s3://{}/{}".format(bucket, file["Key"])
+                    manifest = Datasets.create_json_line(image_path, 0, dttm)
+                    mfile.write(json.dumps(manifest) + "\n")
+
+            # copy local manifest to target S3 location
+            logger.info("Uploading manifest file to %s", manifest_s3_path)
+            response = s3_client.upload_file(
+                output_manifest_file, manifest_bucket, manifest_prefix
+            )
+            # delete local manifest file
+            os.remove(output_manifest_file)
+
+        except ClientError as err:
+            print("S3 Service Error: {}".format(err))
+            raise
+
+        except Exception as err:
+            print(err)
+            raise
+        else:
+            logger.info("Completed manifest file creation and upload.")
+
+    @staticmethod
+    def create_json_line(image, label, dttm):
+        """
+        Creates a single JSON line for an image.
+        param: image: The S3 location for the image.
+        param: label: The label for the image (normal or anomaly)
+        param: dttm: The date and time that the JSON is created.
+        """
+
+        class_name = ""
+
+        if label == 0:
+            class_name = "normal"
+        elif label == 1:
+            class_name = "anomaly"
+        else:
+            logger.exception("Unexpected label value: %s for %s", str(label), image)
+
+            raise Exception(
+                "Unexpected label value: {} for {}".format(str(label), image)
+            )
+
+        manifest = {
+            "source-ref": image,
+            "auto-label": label,
+            "auto-label-metadata": {
+                "confidence": 1,
+                "job-name": "labeling-job/auto-label",
+                "class-name": class_name,
+                "human-annotated": "yes",
+                "creation-date": dttm,
+                "type": "groundtruth/image-classification",
+            },
+        }
+        return manifest
+
+    @staticmethod
+    def delete_dataset(lookoutvision_client, project_name, dataset_type):
+        """
+        Deletes an Amazon Lookout for Vision dataset
+        param: lookoutvision_client: The Amazon Lookout for Vision boto 3 client.
+        param: project_name: The name of the project that contains the dataset that
+        you want to delete.
+        param: dataset_type: The type (train or test) of the dataset that you
+        want to delete.
+        """
+        try:
+            lookoutvision_client = boto3.client("lookoutvision")
```

Comment:
You're already passing this as an argument!

Reply:
Fixed